dango/indexer/httpd/src/server.rs

```diff
@@ -91,9 +91,7 @@ where
     tracing::info!(%ip, port, "Starting dango httpd server");
 
     #[cfg(feature = "metrics")]
-    let metrics = actix_web_metrics::ActixWebMetricsBuilder::new()
-        .build()
-        .unwrap();
+    let metrics = actix_web_metrics::ActixWebMetricsBuilder::new().build();
```

Comment:
@penso I bumped the dependency version, and the `.build()` method is now infallible.